src/main/java/com/minecolonies/coremod/colony/buildings/modules/QuarryModule.java

```diff
@@ -73,12 +73,16 @@ public void onColonyTick(@NotNull final IColony colony)
     }
 
     @Override
-    public void deserializeNBT(final CompoundTag compound)
+    public void deserializeNBT(CompoundTag compound)
     {
         super.deserializeNBT(compound);
 
-        final CompoundTag quarryCompound = compound.getCompound(getModuleSerializationIdentifier());
-        final int[] residentIds = quarryCompound.getIntArray(TAG_MINERS);
+        if (compound.contains(getModuleSerializationIdentifier()))
```

Comment:
I wonder if this common code could be moved to the code that calls this method

Reply:
checked and not really nicely, the caller is the one calling all modules and it doesnt even affect all modules of this type 😅 

Probably doesnt matter since this is just backwards compat code, to be deleted